src/components/CurrentLocation.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+class CurrentLocation extends React.Component {
+
+  render() {
+    const { location } = this.props;
+    const circleStyle = {
+      background: location.color
+    };
```

Comment:
I changed it to a pure function, but I'm unclear how to separate the styling when it is dynamic and relies on a value from `props`.